app/models/patron.rb

```diff
@@ -137,7 +138,19 @@ def checkouts
   end
 
   def fines
```

Comment:
It seems like a patron unit test is probably in order?

Reply:
Good call. I've pushed this up.